samuroi/util/mask_generator/basic_image_processing.py

```diff
@@ -0,0 +1,33 @@
+__author__ = 'stephenlenzi'
+
+import numpy as np
+import cv2
+
+
+def smooth_data(data, iterations=4):
+    """smooth image stack
+    ::param data: NxMxF ndarray, where F is the number of frames and NxM are the image dimensions
+    ::param iterations: number of times to run the smoothing
+    """
+    # function for smoothing a single video
+    sv = np.zeros_like(data)
```

Comment:
this can be simplified and implemented without recursion
sv=data
for i in range(iterations):
 sv[] = ...
return sv